nifi-commons/nifi-record-path/pom.xml

```diff
@@ -66,6 +66,11 @@
             <artifactId>nifi-record</artifactId>
             <version>1.10.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-properties</artifactId>
```

Comment:
Was this dependency actually needed for anything?

Reply:
`StringUtils.isEmpty()` is in this dependency. At my first commit, I used `str == null || str.isEmpty()` but Mike suggests to use `StringUtils.isEmpty()` instead and `nifi-record-path` hasn't, has it?